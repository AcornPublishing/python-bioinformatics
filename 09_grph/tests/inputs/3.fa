>Rosalind_1071
AGATGCCGGACGCGCGCACATATATAGACGCAGTTCGACGTCTCACACGATATTCACTCG
AGCAATAAGTCGCGTCACACTAACCTATGGCGC
>Rosalind_6708
TTCGGGGAAATCAACACCACCTTTTCCTTTGACCACAGGTCGGTGAGAATGCTTAACCAC
TTCCAACAGGCAGGTAGCGAG
>Rosalind_6245
TGCCGTTTACCCAGCGTCGAATTTAAAGGGGTGAGATCGTATGTATGACCGGGGCATTGC
TCTACAGCAGTTGCATTAAGCTG
>Rosalind_3213
TCAATTCAGAAGCACTCGTCCGTCCTGCGAGTTCAACGGGCCTGACGAGGCGAGGCGTCA
ATTTACACCGCTAACCCAGGGCAGGTTGAGGAAGGA
>Rosalind_9143
GGGGGGAATCCGGTCATGAGTCGGATCTCAGCCTTTTGGAATAGTTGAGGCGAATAGCTT
TGAGGATGCACAACCTGAGCT
>Rosalind_6156
GCAAGATCACACTGGCTCTGAGCACATCCGTCGATTAACCAGAAACTCGCAGAATTGCTC
GTGTGCGCACCCGCCCAGGCCAAAGACGA
>Rosalind_4963
TAGGACGATCCGCCTAGATTAGTCGATCATGGATCTATATGAATCTGATGGGGATTAACA
TTAGGTTCATGGATTTCACTCGTAGCTAAGAAC
>Rosalind_9110
CTCGATCGTGCTCGTTAGAAGCTTTTTCACGTGGGTTTGGGTAGCTTGCACGGGTTATAT
CAGATAAGTTAAGCGTTCACATGCG
>Rosalind_3679
CCGGTATAGTTGCGTGCCTCAAACGAGAAGGGAACCTGCAGGACCAATTGTCTCTGTTGG
ACGTGGAGCTCGTCCAGATTTTGATGATGATAGACGTGGT
>Rosalind_6969
GAGCATACCTCCTCAGCGAACTGGAGGTTGCCATTCCTATGGCCGGAAACCGCCTATAAG
CTCCGGTTTTTAGCCCGTAGCCTATTTTCACGATC
>Rosalind_7497
GCCGCCCCCTATTGAACACTTATAGATTCACCGCAATTGATAATCTCGTTTTCTATGGGT
CTCAACGCTAGACGACGAGATAGCATCTCTGATTCCTTTG
>Rosalind_4080
CCGAGGAGAACTTGTTATTGCGTCAAGTTTCCTTCGGCTTGCTGTCATGAGACCCTGGCT
TGTCCGAGACAATTCTAGAGTCCGCTCCCTTC
>Rosalind_9138
TGGTTTGTTCGCCTCTTCCACACTCCCGCGGGCGGATCCGTGGAGACCCACAGTAGGTTC
CCGTGTTCCTAATGCTGCGCACCG
>Rosalind_5365
TGACTTGTTGATTTCTTAACACCAATACAGACGCAGCGCGAAAAGAGTCGTGCAAGGCTG
AACTAATAACTATGCCCTGTACCGAAGCTATTGTCGT
>Rosalind_1078
ATCATGAGGCTTCTCGCCGCGTAAGGATGCCGTGCTAGGACGAACTATATGTCTCGACCG
AAAGAAAACCTACGAGATGC
>Rosalind_2014
TCGCGGCTCTTCTTAACTTTCCCGTGCCGGCGAGCGCCCGTAAAGCCCGGCAGTTGTCGA
GTACACATTAGACCTTCAATTA
>Rosalind_6983
ATCCACTGGACCCCTACCTTGTAAGAGTGAGGATGCCCTTCCCAAGTGAAAGGTGTAAGC
CCATAGGAGTATCCTTTATACCGC
>Rosalind_5581
CTAAGGCGCGTACTACTTAGCTGCGTAAATACGTGAGCGGCTGTCACCACATTTCTCGCA
GGTCTCTGAGTCTATGTCCG
>Rosalind_3545
AATCCCGTCGATTTGTACGCGATCCTACTCCCCTAACGTCCGCCCAATACAATGATAGGG
ATGGCCATCCTTATAATCCTGC
>Rosalind_1083
TCTTACGCAAGAAGCACCTATTACTCGATAGGAGAACTTATATGCGATCACCTTCACTTG
CCTAGCCGGCACAACTAAATACCTCTTCGCGTAT
>Rosalind_1032
AACCGGATAACCCACTTGCCAGTCACATGTCTCTGGTTCTTACCGTGCCGGCTTTTGCCC
CGTGAGACATTTACGCTGCGACTTGTGTATCGCTTGA
>Rosalind_3333
TTAAAAATTTTGGTTGTGGTGGGTTACAATAGGTACAATCCTAGATTAACATGTGGTTAA
GGTGCCAAAAATCATCCGAGGGAG
>Rosalind_7026
AAACAAAACGCACGCTTACACCTAAAGGGGGGTATCAGGTCGTGACGTGAGCATGCGCTT
CGGGCGATGCACGTTTCGAGAGCGGCGAAGCCCA
>Rosalind_2343
TAGCCAGGTGCGACGACCGGCAAAAACTGTCTTCACCTATGGGCAGAGGGGTGTCGAGTA
AACTAGGCAGGGGGTCCTGCATGTTGGTGAGGATGAG
>Rosalind_0610
GAGTAGTTGTCCACCTCAATCTCATTACGTCTGCCCGCCCTCATTTCGTCTCGTTATTAG
AATGACATGTACGGCCATGCTTCG
>Rosalind_6425
TTGACAATACTCATTTCTCGATTTACTCGCAGACCGTAAGTATACTAATTTTGTAGGCAC
TTCCACGGGGGGATGAACCTAAGACCCGC
>Rosalind_5221
GTGAGATCGACATAACCATTTGTACCGCTAAGACACCCTCGATCCGTGTCTTCGCGGGAC
TAATTAACACTTACGCAGGCCCGAGGGTAGA
>Rosalind_9524
TGGTGATTGATCACACCCGTGGAATACACTGTAATGGGAACGGTCGTCCGAGATGCTATC
TAACCGGAGGAACAGGCTGTTGGGGT
>Rosalind_2771
TAAAATATAACGCTGACTCAAGTTAAACTTGATGCAGTGGTGCCCTTTGGGACCTGTATG
CTATCTGAGACCCCGCATTGTCTCTACAACTACAGTTCCT
>Rosalind_8232
ATCCAGCCACTTACACTGCTTTGCCACTTACGATCGGACCCCAGACGGAAAAAAAATGCT
GAATGCGAACTAGTGCTGGTGCTGAGACGCGAGAA
>Rosalind_9056
GTAAGCCAAACCCTCAGGCTCAGGTCCGCTACTACTTTACTGTAAGTCATGCTCAACTGG
AAGGCCATGCGGGAAGCCTCTCTCAGAAGA
>Rosalind_1715
ACTGTCATCAATAGATGTGGAGCATACAGCGCTACCACGACCGTTTGCTTGTGCATTCAC
GTAGGTTGATTTGTTCTGCCC
>Rosalind_9475
TTAACCTGCGACTTGTAACTGTACCGTGTAGCGGTTAGCTCGGTGAGAGCATACTAACTC
ACTTTGAACGTTCGTTCTTGTGGGC
>Rosalind_4012
ACTACGCTTCACTCGAAGACTTCCCCTTACTCGCTAATTCTAGCCGGCGCGTTTTGATAT
CTTGAGCGCAGATCTGAACTGTAATTGGAGGATCACT
>Rosalind_8754
AGGATAATATCGGCATGGTAAGGGTGTCGGCCACATATTTAGCGATGCGTCGGTGAGAAA
ATACGCTCTGCTGAAGATAATGGGAATACAGCGCTG
>Rosalind_6165
CCTCCGTTAGTGATAACCTGCCAGGACGTTGGCGCTGCGAACTCTACTCAGCCCCTGTGT
CCTGCCATGAAATTCGCTATTCGGTGGCCTTGTACGTGC
>Rosalind_8270
GCGGCATAGTAAGCGCCTGCGGAAAGCCTGCTATACTGCCGCGAAACAGCTAAAGTATGA
GGTTCTGATCGATCGCCCCATACAGCTTATA
>Rosalind_8870
TCGTTGATCCAGGCTGGAAAGTTACTACAGATAGGAATACTGCCTATCATCATATGGATT
TAGCACCCGATTCTGGTTCGACACGAACTTCCCC
>Rosalind_4513
TGGTGACTCGTCTAACGCGGAGGTACCACATGGATCTGTCACCTGTATGCAATTGCCAGG
AATAGTACGATCGTGTAAAGTCCTGAG
>Rosalind_4509
CAAATGGGTGTGGCTTATGCTAGACCCTTTACGTACATACCCCTCATTTTTATATACTCG
CCACCTAGCTTTTCCTAGGTTTACCACAGTTCAC
>Rosalind_6938
GAAGAAGTAGGACGAATCTTAACCCTGTGACTCTTTCGAGGGCTCTCCCTAGTGTCTCGT
ATCTAAACCAACCCGATGTACTAGAGGTGGTTAGAGC
>Rosalind_5108
TTGAGGGACATGTGTGCGGAGTCACCCGATACGAATTTGTGCCTGATGCGATCCTAGATA
ACATGTAATTCCACCGACCTTAAAGGGTTAGGGCTT
>Rosalind_8452
CCGCATGTTCTATTGCTGCTGGTGTGTAATCAGCTAAACAAAACACTTATCCGGTAACTT
GCGGCCAACGAGCCTGAGTCCGAGATAGAACTAGACGAC
>Rosalind_4005
TTGGCACTTCTGTCACCTGTATGTGTATATGGCCCACCCGCGCGGGAAAATGGCCGCCAT
CCTTGATTTACTCTCGTACTCGGCTGATCGCTC
>Rosalind_0907
TCCCTGATGTGCAGATGTTCTGTTCTCAATTAAAGGAATAAACGAGCTCTAAAGAGACGC
TTGGCCTACTCATGAGTATG
>Rosalind_1181
GGATTGTTCTATGAGTGCCGCGCCTGCTCTGTTGAGTATCGCGTGATGATTCTTGTCTTG
TGATAGCCTTACCAGGCTCTAA
>Rosalind_2644
CTGATCGTCATTTAATATAGGTAATATGGGCTACTCCCCACCCCGACTCACGTCCCGTGG
GCCAACGTTTAAGGGATACGAT
>Rosalind_3393
CAGGAGCTTCTCATGTTGCCGAGAAACAGTACTAGCTGAGCCAAAATGGTAATTGGATCC
TTTGGCCACCTGATCGCCCCC
>Rosalind_7227
CTTGTATTTAGTGACCCGTTATCGTTGTTGGTCAGCCTGAGGGTAGCTCCACAATAATGT
AGCTCTGCCGTCCCAGAGCGATGGGGACTCTAGCA
>Rosalind_2772
TAACGAACTAATTGACAAAACCAGGAAGAACGGGAGGCTAGGCCATGCAAATGACGGCGA
CCCCATTTCTTAGTCCAAGTTGACGTTACTCT
>Rosalind_4771
CTCCTAATTGGGTTTATAGCGTTCTACGCCCCTCATATGATACGTGGAAAGGTCGTCTGC
GCGGGTTTGACGCCTCTTCTATCGATCA
>Rosalind_7444
CATAGGCCCATTGCGTAACGGCGCAAAGAACATCTACCACCAACCCTTGTTCCTAAGGGA
CTGATTGATCATCAACGCATGTAAACAGCTAAC
>Rosalind_4116
TCAGCCTCAGCATACTGAGCTAAGTTGTACCCTGCTCTAAGGTGCAAAAAGCTCCGTTAG
CAGTCGTGCCAGGTGGAACGCAG
>Rosalind_6895
CTCCCTCAATGGATTACTGGCCGCGGCTAGATCGCCACGCGTCTTTCCCGCGAAGCGAAT
CCGCACCATACCTCCGCATCACTGATGGTCGT
>Rosalind_4292
GGTGGGCGTCTCTCTCCCAACCCGATTGGGCTCCCGACGACCGGCGAAAAAGAGACCCGA
TGTGTAGCTTTTCGACCGTACGAGGCCTGTCAACACCTGC
>Rosalind_1998
CCAATATTACTAAGCATGACACCGACAGCGGAATGCAATGCGCCCCAGGCCTGTTGATAG
AATAATTCCGTAGTCTTGAATACTTGGAGGGGC
>Rosalind_0821
CTCTCAATCCCTAGGTTGAACATTTAACCCCGCTCGCTCATAAAAATGCATCATGTTCAG
ATGACAGCATCACTATTAAGTACCGTCCTGAGGG
>Rosalind_9503
CTGCAGGTGCGAGGCCATCGGAGTAGGCTGGATGGGTGCCCTGTTACCTCAGGTGGTAAG
TGATCGCAATTCCATGCCAGTCGATAGGTATAAA
>Rosalind_6125
AGCTGCTAATCGATTGCATTGACAACACGAAGCCTCACAAACAACTAGTCCTTCTTTAGA
GAATTACCCATAGTGTTGTTACCGC
>Rosalind_7895
GCGAGTCCAGACGTAACGGCCAGCGAAAATCGAGCATTTGGTTGGCGCCAACTACTACAC
ATATACTGTGTCATATCGACATCTACCAATATCCTG
>Rosalind_3662
TAACCCTATAAATACACTCAGAAACGAGCGGGTACAGCTCCAGTGTAATTAGGGAAAAGC
TTTTAACGCCTGAGTAGTACGAACACTTCGGTCGA
>Rosalind_3359
TTGGCCTAGTCTTACAAACCCTGTAGCGGCGACTTACAGGCTCAGTATTGTCCGAGCCAA
ATCTTGCCAACATGGAAGTTCAACGG
>Rosalind_8964
GATATGGAGCCACTTGCGCTGTACACCGATCTAACAAGCTTCTCTGTGCGGGTCGAAGAG
CCAATTCTCAAGGATGGGCCCGGTCCGGAATC
>Rosalind_2307
GACCCGAACGCCAGGCACGCACATCCTCCCTTTGGGTACGAGTAACAGGGTGCGTTGATC
TGTGATCACGTCTTCTCTCGCCGCACGGGGGTAATACGGC
>Rosalind_6131
ACAGCCATCGCGGTTCGAGCGCATATTTGTCCCGATATAAACTCACTAATAGAGTAGTTA
CAACGAGTTATCTCCCCCACCACAATA
>Rosalind_8442
ATGACTGCAAATCCCTTATTCCGCGAGAGACCCGAGGCCGCGGGGATGAGGTCACCTTTG
GCTTGGTTTACCAGCACTGCACGAGCGCCTGTTC
>Rosalind_3975
CAAAGAGAGTGCCTCTGATGGAGTGCGTGGCACCTACAATTTGTCTAGGGGGCATTTTAG
TCGCACAGCCCAGGAAGAGGCAC
>Rosalind_5025
GTTCTCTTACACCTTACTTCTGCCGTACAGCTATCCTGAATCAACGTCGGTCTAGTAGTG
AAATTCCATGGTATTAGTCTAGC
>Rosalind_0040
CATCGCGTTCGGGACGGCGATTTTTGTCGCGCCGTGTTGGCGGCTGAGCCCTAATTGCGC
CAGTGGACTTAATCTGCAGCCTTGTATTTCTACCCATGT
>Rosalind_0467
CGCGAACACGCTCCAGCACACTCCGCTGGACTCCACGAGCCATACTGCATACTAGCGCCG
CCCACAAGTACCGGCAATTTAGAAACTTCTT
>Rosalind_9909
CATTTCTACACAAGGACCCGGTATGTGCAAGGGTAGGGTGAGTGCTTTTTAATCTAGGTT
GCCTTTCACTCGGGCATTTTACCTTTCCAGTTAGTGGT
>Rosalind_9814
AGTCCTACTATCTCGGGAAAAATGTGTCAGCCTTGTGTTTGTATAATAATAGATCGAGAG
CATTGAGGACAGATACTGAAACTTTTTAAAAA
>Rosalind_9093
TCGCTTTCTACTGTGTAATCTCTCCAGCGATACAGTTCTATTCTGCAACACCAAGGCGGC
GTAACACATCAGGCTCAAGTCCCGCGGATGGTA
>Rosalind_2267
TGCACGGTTGGTGAATGCCCGACCGTAGTCGAGGGTCGTCAGTCTGAACTTAGATTTGTA
AAGCCGTACGTATGGGGCAAACCC
>Rosalind_2036
CTCCGGTCACGACCATTCCTCCGTTTGCAAATGGGCAAATGCAAGTGAGACTGGCCTTAA
TGGGGTAAACGCGACGGGGCCAACACGTCGAC
>Rosalind_4585
GTCCGAGAACCCCCGCCGTTTCGGGTCAATTTAGTTAGACGAGAGCGCGAACGGGCCTCC
GGGTAAATGTCATCGGACTTTG
>Rosalind_2350
CGTCTCGATAGAAGAAATGGTGTCGGGTCGCGTCTAACGTAAGGACACCGAGTTCCATGG
AAGATCCGCTCGATCGTCAACTATACTCATCG
>Rosalind_8935
TGAGTATGCGACCATTGTCCGCGCGGATCAGCGGTCGGCAGAGAGACCGTAGGTTATTGA
CAATGCACGGCATAGGGGGCTTACCACG
>Rosalind_4964
ATCGCCGCTCTTTAGAGACACCGCCTCTCGTCATCACCACCCAACGACTGGATCTCCACC
AAAAGATTTAAGAATGGTGTCTGAAAC
>Rosalind_9047
GTATAGCCTGGTGATCTACCTTCATGCGGGTGCTCACGGGGATTGATCGATGTTAACGAC
AAGCCAATTAATAAGGTTCACGAGAGAAT
>Rosalind_8162
GATCTCGTAGCGAATAGGATCTTGGGGTTCCTTGCGGCGTAGGAATCGGGGTGAGGCTAA
CCATTTACAATTACCTCAGCATGGGTGCCTCATGCGG
>Rosalind_4227
ACCCATGGCTCTTTCTGTCATAAGGATGTTAGCGGTCATCCACCATTAGCTGTGCTCTAG
CTAAAGCGTCTCTCACAATT
>Rosalind_7960
CAATATTCGGCTGGACCTTGTCGTAATAACCTCTCTCGGTCAGTAGTGCTTTTCACAGAG
CACAACCTAGGAGCGTCCAAGGAACG
>Rosalind_9039
GGGCTGGGCGGCTCATCTCGGTTTATGGCCGTTCCCGTATCGACCCCGTATGATGTCTAT
CAAGGATGTTCTAGTATGAAAGGGCACGGGG
>Rosalind_0070
TTCATTTGCTATATCCTTGAAATCCTTACGAGCTCTATCCTGGACGGCATCTGGAAGGCG
TGCCTCGAAAAAAGGCAATCTAGACTGT
>Rosalind_1532
ATCTGATAAACTACTTTTGAGACATGGCTGCATCAACGCTGGGGATTTGCCGCTACCGTC
AGTTCAGAGGTGTCTTGGGAGCGGCAGTCG
>Rosalind_4407
TATACCTCCCGTTATCTATCATCTGCTGCTCGGCGGCGGTAAGCTATACAGGCCGAAGCG
ACAGCAAAAGCGTCCGGTCGAATAGGGTATCATC
>Rosalind_9549
CATTTGAGAGAATTTTTACGAGAAGACCCTCCGACTCCTATATGGCTGACCGATAAGAAG
ATACTCACTCTCACTGCCACATCCTTCCCCCGCATACG
>Rosalind_8786
GCAGTACAGGTTAGTCGAGAGTGGGGTTAACAATAGGTCGCAGGGCTTTGCAATTATACG
AGACGAACTCCACGCCCTGATT
>Rosalind_7170
TACTTCACAATAGGCATTAAAGTGGGAATTTTGGCGAGGTACAATAGCGAAGGAAGTCTC
ATTGCAGACCCTAACCCTAATAGGCATT
>Rosalind_2102
GAAACATCGAGCGTGGGCGACCCTTCTTAAGTGGTATGTGTGATTGATAAACGTCTCCAA
GACAATTTCGGTTTAGATCAATCACTTGAGCA
>Rosalind_3612
GAGTAGGTTTTCAACGCTATAAACCATTAGCAGGCGGCGGATTAGGAAGTCTGCTTAGCT
GCGAAGTTTTATCGGTTGTCCTGTCATT
>Rosalind_1809
CATACGTCGAGCCTTCCTATTAGAGATGTCCACGGCGCGCGTGATTTAGAGATGTCTCGA
TTTCCTGCCCACTAGGTGTCGGCTA
>Rosalind_5201
GACTATTGTCGGTATAGTTCTTGGTCGCGAGTCCTCGGAATCGACAACAGTGTGAGTAAC
GACGAATTTACAATCTCGGCGATATGTCAACA
>Rosalind_1489
CCGGCTCGATATCATTCACAAGCAGGTTAGCCCGCCAAAGTCAGCAAATTTTCGGATCCC
TTTTGTAATCTGTGGGATTCGTC
>Rosalind_1865
CGTGGGTAAGCTCCGAACATCCAGACGCATTAATTCTCATTGATTATGCTGGAAAAGTGA
CACGCTCGACAGCTTTCAGCTCGTTCCT
>Rosalind_3260
TGACCACATAATAATTTGATTAGGAAAATTGCTGTACTCCATTAGGTTTGGAGAATGTGC
CCAAAAGACAATGGAAAGGTAATTATCT
>Rosalind_6724
GGTTAGCGTCATTGAAGCAGACACAATATAGGGACCGCATTATTCACACCGTATGGTCAA
GGTAACGTTTCGGATGTATCGA
>Rosalind_9595
AGTAATCATGCACCTGAGCTTGATACTCGCGGATATTGAAACTCCTGCCGCAGCACGGAA
CGTCCTATGTAACGGGCTTCCGGTGA
>Rosalind_5627
TCAACGATCTTGCTAAGCGGCCTAGGCCGGAATCGCGATGGTTGCCACACTACGGAGCAA
TGGTCGCTCCAGGCACCCCATTCTTAATAGG
