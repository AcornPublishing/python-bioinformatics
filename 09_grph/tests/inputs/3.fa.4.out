Rosalind_0467 Rosalind_1083
Rosalind_0610 Rosalind_6708
Rosalind_1032 Rosalind_5108
Rosalind_1032 Rosalind_6425
Rosalind_1083 Rosalind_9047
Rosalind_1181 Rosalind_5581
Rosalind_1489 Rosalind_2350
Rosalind_1998 Rosalind_9039
Rosalind_2267 Rosalind_4227
Rosalind_2343 Rosalind_8935
Rosalind_2350 Rosalind_4964
Rosalind_2772 Rosalind_0821
Rosalind_3213 Rosalind_8754
Rosalind_3260 Rosalind_1532
Rosalind_3545 Rosalind_9503
Rosalind_3612 Rosalind_9549
Rosalind_3612 Rosalind_9909
Rosalind_3679 Rosalind_4513
Rosalind_3679 Rosalind_9138
Rosalind_3679 Rosalind_9524
Rosalind_4116 Rosalind_8786
Rosalind_4292 Rosalind_9503
Rosalind_4407 Rosalind_0040
Rosalind_4513 Rosalind_8935
Rosalind_4771 Rosalind_1078
Rosalind_4964 Rosalind_7026
Rosalind_5025 Rosalind_2343
Rosalind_5365 Rosalind_8870
Rosalind_5627 Rosalind_4963
Rosalind_6125 Rosalind_8452
Rosalind_6425 Rosalind_8452
Rosalind_6895 Rosalind_8870
Rosalind_6938 Rosalind_6969
Rosalind_6969 Rosalind_8162
Rosalind_6983 Rosalind_8452
Rosalind_7170 Rosalind_9549
Rosalind_7170 Rosalind_9909
Rosalind_7444 Rosalind_2772
Rosalind_7444 Rosalind_3662
Rosalind_8162 Rosalind_8270
Rosalind_8270 Rosalind_4407
Rosalind_8442 Rosalind_5025
Rosalind_8964 Rosalind_3545
Rosalind_9039 Rosalind_9143
Rosalind_9143 Rosalind_6125
Rosalind_9475 Rosalind_9039
Rosalind_9503 Rosalind_2771
Rosalind_9595 Rosalind_5221
Rosalind_9909 Rosalind_4513
Rosalind_9909 Rosalind_9138
Rosalind_9909 Rosalind_9524
